### ISSUE TITLE
No update on too small network latency

### ISSUE THREAD

**Issue by MeinAccount** [wbbaddons/pushNotification#4]
The keep alive-request is ignored when the notification was created in the same second as the request is being processed. See [`SessionAction`](https://github.com/WoltLab/WCF/blob/master/wcfsetup/install/files/lib/data/session/SessionAction.class.php#L51).

**Comment by TimWolla** [comment]
Yeah, we already noticed as well (also happens when multiple browser windows are open).

**Comment by MeinAccount** [comment]
This could be fixed by duplicating the method without the return statement. The new method should call the same events as the original calls.
